var/spack/repos/builtin/packages/py-metomi-isodatetime/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyMetomiIsodatetime(PythonPackage):
+    """Python ISO 8601 date time parser and data model/manipulation utilities."""
+
+    homepage = "https://github.com/metomi/isodatetime"
+    url = "https://github.com/metomi/isodatetime/archive/refs/tags/3.0.0.tar.gz"
```

Comment:
https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github

Reply:
@LydDeb, I guess Adam wants you to change the package to pypi:

https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github